docs/transactions.md

```diff
@@ -1211,16 +1211,16 @@ The account is not removed until sufficient number of Trustees approve it.
 - Number of required approvals: 
     - 2/3 of Trustees
                       
-#### UNJAIL_VALIDATOR_NODE
-**Status: Not Implemented**
+#### ENABLE_VALIDATOR_NODE
+**Status: Implemented**
 
-Approves unjail of the Validator node from jailed state and returning to the active validator state. 
+Approves enable of the Validator node from disabled state and returning to the active validator state. 
 
-If more than 1 Trustee approval is required to unjail a node, the node still
-will be in a jailed state until sufficient number of approvals is received.
+If more than 1 Trustee approval is required to enable a node, the node still
+will be in a disabled state until sufficient number of approvals is received.
 
-If 1 Trustee approval is required to unjail a nod or sufficient number of approvals is received,
-the node will be unjailed and returned to the active validator set.
+If 1 Trustee approval is required to enable a node or sufficient number of approvals is received,
+the node will be enabled and returned to the active validator set.
```

Comment:
The description is incorrect. There is no voting process for this command. Also please specify which user is authorized to send this transaction.

Reply:
fixed